### PR TITLE
header26

### PR DESCRIPTION
**Pull request by yusufktlk**

### CODE REVIEW

File: editor-components/header/header26/header26.module.scss [hunk 1]
```diff
@@ -1,150 +1,195 @@
...
+        transform: scale(0.9);
       }
+
+      // &.shrink {
+      //   transform: scale(0.9);
+      // }
+  
+      // &.expand {
+      //   transform: scale(1);
+      // }
```
Comment (Ayazelar) [inline editor-components/header/header26/header26.module.scss:87]: remove unused codes

File: editor-components/header/header26/header26.module.scss [hunk 2]
```diff
@@ -1,150 +1,195 @@
...
-        flex: 2;
 
-        img {
-          width: 100%;
-          height: 100%;
-          margin: 0;
-          margin-left: 0px;
+          .arrows {
+            position: relative !important;
+            // flex-direction: row !important;
```
Comment (Ayazelar) [inline editor-components/header/header26/header26.module.scss:121]: remove

File: editor-components/header/header26/header26.tsx [hunk 3]
```diff
@@ -1,120 +1,246 @@
...
 
   getName(): string {
     return "Header-26";
   }
 
+  handleBeforeChange = () => {
+    if (this.sliderRef.current) {
+      const slides = this.sliderRef.current.innerSlider.list.querySelectorAll(`.${styles.sliders}`);
+      slides.forEach((slide: HTMLElement) => {
+        slide.classList.add(styles.shrink);
```
Comment (Ayazelar) [inline editor-components/header/header26/header26.tsx:150]: use decorate css method

**Review by Ayazelar — CHANGES_REQUESTED** [review]
.

File: editor-components/header/header26/header26.tsx [hunk 4]
```diff
@@ -1,120 +1,281 @@
...
-                <h3 className={this.decorateCSS("items-description")}>{items["left-top-item"].description}</h3>
-              </div>
-
-              
-                  <ComposerLink path={button.url}>
-                    <div className={this.decorateCSS("button-wrapper")}>
-                      <a
-                        className={this.decorateCSS("button")}
+          <ComposerSlider {...settings} ref={this.sliderRef}>
+            {this.castToObject<[]>("sliders").map((item: any, index: number) => (
```
Comment (dogusmiuw) [inline editor-components/header/header26/header26.tsx:235]: after creating the relevant type, cast it to that.
e.g. `this.castToObject<SliderItem[]>("sliders").map((item: SliderItem, ...`

File: editor-components/header/header26/header26.tsx [hunk 5]
```diff
@@ -1,120 +1,281 @@
...
+                <div className={this.decorateCSS("slider")}>
+                  <div className={this.decorateCSS("left-side")}>
+                    <ComposerLink
+                      key={`hdr-32-${index}`}
+                      path={item.url}
+                    >
+                      <h1 className={this.decorateCSS("title")}>
+                        {item.title}
+                      </h1>
+                    </ComposerLink>
```
Comment (dogusmiuw) [inline editor-components/header/header26/header26.tsx:246]: if item.title is empty, do not render this. do this operation for all props.
[please check](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-check-if-a-string-prop-is-empty)

File: editor-components/header/header26/header26.tsx [hunk 6]
```diff
@@ -1,120 +1,281 @@
...
-            <div className={this.decorateCSS("right")}>
-              <img src={items.rightImage} />
-            </div>
-          </div>
+                  </div>
+                  <div className={this.decorateCSS("right-side")}>
+                    <img
+                      className={this.decorateCSS("image")}
+                      src={item.image}
+                      alt="Slider Image"
```
Comment (dogusmiuw) [inline editor-components/header/header26/header26.tsx:268]:
```suggestion
                      alt={this.castToString(item.title)}
```

**Review by dogusmiuw — CHANGES_REQUESTED** [review]
please handle changes

**Review by Ayazelar — APPROVED** [review]

**Review by dogusmiuw — APPROVED** [review]

**Pull request merged**